stable/aws-pod-identity-webhook/values.yaml

```diff
@@ -0,0 +1,24 @@
+tls_secret_name: pod-identity-webhook
+annotation_prefix: eks.amazonaws.com
```

Comment:
As you alluded to, EKS already installs the pod identity webhook by default, so this EKS-specific annotation prefix is probably not appropriate as the default for this chart?

Reply:
Happy to change it but I don't know what for?